docs/source/user/installing.rst

```diff
@@ -185,18 +185,28 @@ vary with target operating system and hardware. The following lists them all
   * ``numpy``
   * ``llvmlite``
   * Compiler toolchain mentioned above
-  * OpenMP C headers and runtime libraries compatible with the compiler
-    toolchain mentioned above and accessible to the compiler via standard flags
-    (Linux, Windows).
 
-* Optional build time:
-
-  * ``llvm-openmp`` (OSX) - provides headers for compiling OpenMP support into
+* Build time toggles:
+  * ``NUMBA_NO_OPENMP=1``
+    DEFAULT: undefined
+    set the environment variable to ``1`` when building using ``setup.py``
+    to disable compilations of OpenMP threading backend.
+    If enabled you need OpenMP C headers and runtime libraries compatible
+    with the compiler toolchain mentioned above and accessible to the
+    compiler via standard flags (Linux, Windows)
+    * ``llvm-openmp`` (OSX) - provides headers for compiling OpenMP support into
     Numba's threading backend
-  * ``intel-openmp`` (OSX) - provides OpenMP library support for Numba's
+    * ``intel-openmp`` (OSX) - provides OpenMP library support for Numba's
     threading backend.
-  * ``tbb-devel`` - provides TBB headers/libraries for compiling TBB support
-    into Numba's threading backend
+  * ``NUMBA_NO_TBB=1 TBBROOT=/usr``
+    DEFAULT: both undefined
+    set the environment variable ``NUMBA_NO_TBB`` to ``1`` when building using
+    ``setup.py`` to disable the compilation of the TBB threading backend.
```

Comment:
I'm unsure as to what other ways there are to build other than with `setup.py` being invoked at some point?

Reply:
removed this wording.